tests/zenko_e2e/backbeat/Using.md

```diff
@@ -32,9 +32,14 @@ export ZENKO_BACKBEAT_SECRET_KEY=<secret-key>
 ```
 export AWS_S3_BACKBEAT_ACCESS_KEY=<access-key>
 export AWS_S3_BACKBEAT_SECRET_KEY=<secret-key>
-export AWS_S3_BACKBEAT_BUCKET_NAME=<destination-bucket-name>
-export AWS_DESTINATION_LOCATION=<destination-location-name>
-export AWS_SOURCE_LOCATION=<source-location-name>
+export AWS_S3_BACKBEAT_BUCKET_NAME=<destination-aws-bucket-name>
```

Comment:
instead of having all the exports examples here, the step can be to edit the example file (which can be updated with the examples) and `source .secrets.env`

https://github.com/scality/Zenko/blob/master/tests/.secrets.env.example
https://github.com/scality/Zenko/blob/master/tests/.env

Reply:
Good idea—this will help us align with pre-existing tests and help eventually with automation. I've updated to use any pre-existing env variables and updated those files accordingly.